src/com/nutrons/nu17/RobotMap.java

```diff
@@ -3,10 +3,10 @@
 public class RobotMap {
 
 	// Ports of wheels
-	public static final int FRONT_LEFT = 0;
-	public static final int FRONT_RIGHT = 1;
-	public static final int BACK_LEFT = 2;
-	public static final int BACK_RIGHT = 3;
+	public static final int LEFT_DRIVE_MOTOR_A = 0;
```

Comment:
Giving the direction "front, center, back" is clearer when translating this into wiring.